test/unit/discovery_taxonomy_extensions_test.rb

```diff
@@ -1,6 +1,9 @@
 require 'test_plugin_helper'
 
 class DiscoveryTaxonomyExtensionsTest < ActiveSupport::TestCase
+  include FactImporterIsolation
+  allow_transactions_for_any_importer
```

Comment:
Did you see failing tests or why this is needed? Let's probably add this in another PR then?

Reply:
yes you have failing tests, because you create facts on an open transaction